sample.txt

```diff
@@ -1,4 +1,4 @@
-This is sample text.
+This is somple text.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/litmon/danger-sample/blob/2658ca5123f31b3a925620ee17fb5a2c6990e6b2/sample.txt#L1"></span>It is typo</td>
    </tr>
  </tbody>
</table>